editor.html

```diff
@@ -80,13 +80,13 @@ <h2><i class="fa fa-upload"></i> <strong>{{ load-title }}</strong></h2>
             </div>
             <div class="load-form">
                 <form id="load-form-form">
-                    <input type="file" name="load-form-file-upload"/><br/>
+                  <input type="file" style="display: none" name="load-form-file-upload" id="file-upload-input">
                     <input type="submit" value="{{ submit }}"/>
                 </form>
             </div>
             <div id="load-alt">
                 <p><a href="#" class="load-form load-toggle">{{ toggle-drop }}</a></p>
-                <p><a href="#" class="load-drag-target load-toggle">{{ toggle-file }}</a></p>
+                <p><a href="#" id="file-upload-link" class="load-drag-target load-toggle" load-toggle">{{ toggle-file }}</a></p>
```

Comment:
The text at the end of the `a` element doesn't look quite right:
```js
< ... class="load-drag-target load-toggle" load-toggle">
```

It's possible the last ` load-toggle"` can be removed?

Reply:
Yep. have done